tests/performance/local/CMakeLists.txt

```diff
@@ -34,93 +15,24 @@ if(NOT HPX_WITH_CUDA_COMPUTE)
   set(stream_FLAGS CUDA)
```

Comment:
Couldn't point the specific line but there is still a `HPX_WITH_CUDA_COMPUTE` (two lines above)